src/search.ts

```diff
@@ -21,15 +21,16 @@ const defaultEffects: SearchIndexEffects = {logger: console};
 const indexOptions = {
   fields: ["title", "text", "keywords"],
   storeFields: ["title"],
-  processTerm(term) {
-    return term.match(/\p{N}/gu)?.length > 6 ? null : term.slice(0, 15).toLowerCase(); // fields to return with search results
+  processTerm(term: string) {
+    return (term.match(/\p{N}/gu)?.length ?? 0) > 6 ? null : term.slice(0, 15).toLowerCase(); // fields to return with search results
```

Comment:
I'm moving this comment to line 22 (the issue predates this PR, I just happened to see it here).